### ISSUE TITLE
MAL-Sync only works on crunchyroll for firefox

### ISSUE THREAD

**Issue by Jazzie7077** [MALSync/MALSync#551]
**Describe the bug**

On firefox, with all other extensions disabled and mode set to Anilist the menus, syncing and shortcut keys only work on Crunchyroll, I attempted this with the exact same settings on Animelab and Netflix, and the menus, syncing and shortcut keys did not work. However, I tried using the extension on chrome and everything worked perfectly.
**Steps to reproduce the behavior:**
Instal MAL-Sync on firefox, set mode to anilist and authenticate, go to crunchyroll.com and find that everything works, but on other websites it does not note: I only tested netflix.com and animelab.com
**Additional information:**
 - Browser: firefox
 - Type: Extension

**Comment by lolamtisch** [comment]
Sorry for the late response. Was busy last week. 
Still dont know what the problem is. Still did not have the time to look into it.

But there is a workaround for this. Just add the pages in the custom domain section, can be found in the settings. I could not test animelab on the video page and for nextflix i dont have an account. But it should work.
![image](https://user-images.githubusercontent.com/12599334/103516612-21300c00-4e71-11eb-9a3c-c099c8f31d0e.png)

**Comment by Jazzie7077** [comment]
Hey, dont worry bout time its a great application and i cant imagine the amount of time it takes, i just wanna help you make it better if i can:
-I've tried adding these sites to the custom domains, and giving the extension all website data permissions, to no avail. 
-I reinstalled the extension and restarted firefox after that did not work either.
-Just to make sure i turned off every other extension and restarted fire fox to no avail.
-I also made sure that animelab and netflix work when the extention is used in chrome, which it did (i used chrome canary for this) and it worked.
